### ISSUE TITLE
empty table should have role="presentation" per WCAG accessibility guidelines

### ISSUE THREAD

**Issue by tailcat** [wenzhixin/bootstrap-table#7415]
### Description

Line 115 of src/bootstrap-table.js has:

 <div class="fixed-table-header"><table></table></div>

The table should have role="presentation" per WCAG guidelines.

https://webaim.org/standards/wcag/checklist#sc1.3.1

**Comment by wenzhixin** [comment]
What is the purpose of this attribute? Could you give an example?

**Comment by tailcat** [comment]
<table role="presentation"> this tells a screen reader to not think of this as a table of things but a table just used for its positioning properties.

**Comment by UtechtDustin** [comment]
Makes perfect sense, no question needed.  
Created a PR for that :)
